### PR TITLE
Added italian translation

### PR DESCRIPTION
**Pull request by TheZal**
Added the italian translation of the messagges.properties file #414

### CODE REVIEW

File: src/main/resources/i18n/messagges_it_IT.properties [hunk 1]
```diff
@@ -0,0 +1,284 @@
+EMPTY_LABEL=
+
+# Exception Code Labels
+INVALID_END_POINT=Il seguente EndPoint {0} non è disponibile. Controllare il nome dell'endpoint da tutte le chiamate EndPoint e riprovare.
```
Comment (preetkaran20) [inline src/main/resources/i18n/messagges_it_IT.properties:4]: can you please fix ^M here?

Comment (TheZal) [reply src/main/resources/i18n/messagges_it_IT.properties]: Now should be ok, sorry

File: src/main/resources/i18n/messagges_it_IT.properties [hunk 2]
```diff
@@ -0,0 +1,284 @@
+EMPTY_LABEL=
```
Comment (preetkaran20) [inline src/main/resources/i18n/messagges_it_IT.properties:1]: please rename the file as messages_it_IT.properties.

there is an extra `g`

Comment (TheZal) [reply src/main/resources/i18n/messagges_it_IT.properties]: Done, sorry

**Review by preetkaran20 — APPROVED** [review]
LGTM, thank you !!!

**Comment by TheZal** [comment]
Thank you! Can you add the hacktoberfest-accepted label?

**Comment by preetkaran20** [comment]
@TheZal  https://twitter.com/sasan_karan/status/1581005971790626821

**Pull request merged**